core/src/test/java/com/graphhopper/GraphHopperTest.java

```diff
@@ -98,12 +98,12 @@ public void setup() {
 
     @ParameterizedTest
     @CsvSource({
-            DIJKSTRA + ",false,511",
-            ASTAR + ",false,256",
-            DIJKSTRA_BI + ",false,228",
-            ASTAR_BI + ",false,142",
-            ASTAR_BI + ",true,41",
-            DIJKSTRA_BI + ",true,48"
+            DIJKSTRA + ",false,708",
+            ASTAR + ",false,363",
+            DIJKSTRA_BI + ",false,346",
+            ASTAR_BI + ",false,192",
+            ASTAR_BI + ",true,42",
+            DIJKSTRA_BI + ",true,47"
```

Comment:
I don't understand this yet. Why do we explore more nodes now that there are more (inaccessible) edges?

Reply:
Yes, I did this check. When we add this line:

```java
 hopper.getReaderConfig().setIgnoredHighways(Arrays.asList("footway", "cycleway", "pedestrian", "path", "steps"));
```

to this test we get: 516,259,232,144,38,46 for the numbers of visited nodes. So there is still a minor difference, but the performance won't be impacted (but only if these highway types are excluded).

Reply:
And not sure where the difference comes from. Is there anything else we could exclude to make it equal?